tests/rados/test_rados_preempt_scrub.py

```diff
@@ -18,210 +19,236 @@
 def run(ceph_cluster, **kw):
     """
     CEPH-83572916 - Verify that the preempted messages are generated at OSD logs during scrubbing
-    1.Create pool and write data in to the pool
-    2.set the osd_shallow_scrub_chunk_max and osd_scrub_chunk_max values to 250
-    3.Start writing the data into pool and start scrub parallely
-    4.Once the scrub started check the OSD logs contain the-
-       - head preempted
-       - WaitReplicas::react(const GotReplicas&) PREEMPTED
-    5. Once the tests are complete delete the pool and remove the set values of the parameter
+    1.Create pool with 1 PG with the pg_autoscaler off
+    2.Create  objects
+    3. Set the following paramters-
+        3.1  osd_scrub_sleep ->0
+        3.2  osd_deep_scrub_keys ->1
+        3.3  osd_scrub_interval_randomize_ratio ->0
+        3.4  osd_deep_scrub_randomize_ratio -> 0
+        3.5  osd_scrub_min_interval -> 10
+        3.6  osd_scrub_max_interval -> 2000
+        3.7  osd_deep_scrub_interval -> 600
+        3.8  min_size -> 2
+        3.9  debug_osd -> 20/20
+    4. Rewrite the data into the four objects
+    5. Search the "preempted" string in the active OSDs
+    6. Once the tests are complete delete the pool and remove the set values of the parameter
     """
 
     log.info(run.__doc__)
     config = kw["config"]
     cephadm = CephAdmin(cluster=ceph_cluster, **config)
     rados_object = RadosOrchestrator(node=cephadm)
     mon_obj = MonConfigMethods(rados_obj=rados_object)
-    ceph_nodes = kw.get("ceph_nodes")
     installer = ceph_cluster.get_nodes(role="installer")[0]
-    pool_target_configs = config["verify_osd_preempt"]["configurations"]
 
+    log_lines = ["preempted"]
+    pool_name = config["pool_name"]
     try:
-        scrub_status_flag = False
-        # Before starting the tests check that scrubbing is not progress in the cluster
-        time_end = time.time() + 60 * 15
-        while time.time() < time_end:
-            scrub_status = check_scrub_status(rados_object)
-            if scrub_status is False:
-                scrub_status_flag = True
-                log.info("Scrubbing is not in progress.Continuing the test execution")
-                break
-            log.info("Waiting for the scrub to complete")
-            time.sleep(30)
-
-        if scrub_status_flag is False:
-            log.error(
-                "Scrubbing is in progress cannot continue the further execution of test"
-            )
+        # enable the file logging
+        if not rados_object.enable_file_logging():
```

Comment:
these command execution were not found in the logs attached. please check. 

http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-S24GB4/Preempt_scrub_messages_checks_0.log

Reply:
Please check the log the following log-
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-YBZHRL/Preempt_scrub_messages_checks_0.log